packages/s3-utils/src/validators.ts

```diff
@@ -1,5 +1,4 @@
-import Ajv from "ajv";
-import type { JSONSchemaType } from "ajv";
+import Ajv, { type JSONSchemaType } from "ajv";
```

Comment:
Huh!  I didn't know linters were opinionated about combining type and non-type imports into a single line.